praw/models/front.py

```diff
@@ -1,18 +1,19 @@
 """Provide the Front class."""
-from typing import Iterator, TypeVar, Union
+from typing import TYPE_CHECKING, Iterator, Union
```

Comment:
Every time `TYPE_CHECKING` is imported, it should be alphabetized. It's a little repetitive for me to keep commenting on the same issue so I'll leave the rest

```suggestion
from typing import Iterator, TYPE_CHECKING, Union
```

Reply:
See my other comment